src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
-    public static SessionFactory getSessionFactory() {
-        return null;
+    private static final SessionFactory SESSION_FACTORY_INSTANCE = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
```suggestion
    private HibernateUtil() {
```

Reply:
not fixed, it's a good practice to prevent instantiation of utility classes - we only need static access here